common/djangoapps/student/views/dashboard.py

```diff
@@ -759,3 +770,22 @@ def student_dashboard(request):
     response = render_to_response('dashboard.html', context)
     set_user_info_cookie(response, request)
     return response
+
+
+def _get_preferred_order_by_course_enrollments(order_by, course_enrollments):
```

Comment:
Mostly this function is sorting, so the name should probably start with "sort_....".  How about "sort_enrollments"?

Reply:
There should be tests for this, no?